src/index.html

```diff
@@ -13,9 +13,46 @@
       "
     >
     <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__logo"
+        src="./images/imac.jpeg"
+        alt="27 inch Apple iMac Retina 5K"
+      >
+
+      <h2 class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__id">Product code: 195434</p>
+
+      <div class="card__feedback">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card_review">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__cost">Price:</p>
```

Comment:
`p` is not a correct tag here because semantically `p` means a complete piece of information but `Price:` is just a part of a message

So use `p` for entire block but here you need a `span`